### PR TITLE
3193 Update Understanding Focus Not Obscured

### LINKED ISSUE #3193: Focus Not Obscured (Enhanced) should permit opaque overlays
> Should (Enhanced) version be so tolerant of opaque overlay?

Actually, @bruce-usab, on re-reading the Enhanced normative text, I do not believe that a lightbox should be considered to fail this criteria. The language does not use "obscured", it uses "hidden".

> no part of the component is hidden by author-created content.

I don't think we can make the case that dimming something hides it. Even in the minimum version, we don't say the lightbox is failing Focus Not Obscured. We say it is likely to fail 2.4.11 Focus Appearance.

On rereading that part of the Minimum understanding document, I have some changes to suggest.
1. Given that 2.4.11 is now a AAA, my feeling is that the language in the minimum should actually point to [Non-text contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html), and 
2. the Enhanced version should point to [Focus Appearance](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html).

That gives us something appropriate to assess at each of the levels. If you agree, can you alter this PR to support that?

As part of that, you will need to remove this commit: https://github.com/w3c/wcag/pull/3163/commits/61dcd0c6face13c2581e32753f45815092334daa

At that point, the language for Enhanced seems fine for lightbox (assuming the link points to the AAA version of Focus Appearanced). The AA version would need to be updated.

_Originally posted by @mbgower in https://github.com/w3c/wcag/issues/3163#issuecomment-1542499814_

### PR DESCRIPTION
**Pull request by bruce-usab**
This is a placeholder PR pending PR #3163 being pulled into the editors draft.

### CODE REVIEW

File: understanding/22/focus-not-obscured-minimum.html [hunk 1]
```diff
@@ -17,66 +17,66 @@ <h2>Status</h2>
...
-         <li>People with limited or low vision, who may primarily user a pointer for screen orientation and repositioning, nonetheless benefit from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall viewing portion of the screen.</li>
-         <li>People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to discover where the focus is located.</li>
+         <li>Sighted users who rely on a keyboard interface to operate the page will be able to see the component which gets keyboard focus. Such users include those who rely on devices which use the keyboard interface, including speech input, sip-and-puff software, on-screen keyboards, scanning software, and a variety of assistive technologies and alternate keyboards.</li>
+        <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall useable portion of content.</li>
```
Comment (dbjorge) [inline understanding/22/focus-not-obscured-minimum.html:48]: missing word + correction to american spelling convention:

```suggestion
        <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) benefit from a visible indication of the current point of keyboard interaction, especially where magnification reduces the overall usable portion of content.</li>
```

File: understanding/22/focus-not-obscured-minimum.html [hunk 2]
```diff
@@ -91,7 +91,7 @@ <h3>Additional Techniques (Advisory)</h3>
          <h3>Failures</h3>
          <ol>
             <li>
-
+              An interaction that causes content to appear over the component with keyboard focus, visually covering the keyboard focus indicator. This behavior might be encountered with advertising or promotional material meant to provide more information about a product as the user navigates through a catalogue.
```
Comment (alastc) [inline understanding/22/focus-not-obscured-minimum.html:94]:
```suggestion
              An interaction that causes content to appear over the component with keyboard focus, visually covering the keyboard focus indicator. This behavior might be encountered with advertising or promotional material meant to provide more information about a product as the user navigates through a catalogue.
```
```suggestion
              An interaction that causes content to appear over the component which is getting keyboard focus, visually covering the focused element. (Future technique)
```

File: understanding/22/focus-not-obscured-enhanced.html [hunk 3]
```diff
@@ -21,16 +21,16 @@ <h2>Focus Not Obscured (Enhanced) Success Criterion text</h2>
...
+  <p>Another form of obscuring can occur where light boxes or other semi-opaque effects overlap the item with focus. This form of obscuring is _not_ in scope for this Success Criterion. While less than 100 percent opacity is not causing the component to be <q>hidden</q>, such semi-opaque overlaps may cause a failure of <a href="../understanding/focus-appearance.html">2.4.11 Focus Appearance</a>. When a focus indicator can be covered by a semi-opaque component, the ability of the focus indicator to pass 2.4.11 should be evaluated (and pass) while the focus indicator is under the semi-opaque component. The intention in both situations is that the component receiving focus should never be obscured to the point a user cannot tell which item has focus.</p>
```
Comment (alastc) [inline understanding/22/focus-not-obscured-enhanced.html:26]:
```suggestion
  <p>Another form of obscuring can occur where light boxes or other semi-opaque effects overlap the item with focus. This form of obscuring is <em>not</em> in scope for this Success Criterion. While less than 100 percent opacity is not causing the component to be <q>hidden</q>, such semi-opaque overlaps may cause a failure of <a href="../understanding/focus-appearance.html">2.4.11 Focus Appearance</a>. When a focus indicator can be covered by a semi-opaque component, the the focus indicator should be assessed against 2.4.11. The intention in both situations is that the component receiving focus should never be obscured to the point a user cannot tell which item has focus.</p>
```

File: understanding/22/focus-not-obscured-enhanced.html [hunk 4]
```diff
@@ -21,16 +21,16 @@ <h2>Focus Not Obscured (Enhanced) Success Criterion text</h2>
...
   <ul>
     <li>Sighted users who rely on a keyboard interface to operate the page will be able to see the component which gets keyboard focus. Such users include those who rely on devices which use the keyboard interface, including speech input, sip-and-puff software, on-screen keyboards, scanning software, and a variety of assistive technologies and alternate keyboards.</li>
-    <li>People with limited or low vision, who may primarily use a pointer for screen orientation and repositioning, nonetheless benefit from a fuller visible indication of the current point of keyboard interaction, especially where magnification reduces the overall viewing portion of the screen.</li>
-    <li>People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to more easily discover where the focus is located.</li>
+    <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) from a clearly visible indication of the current point of keyboard interaction, especially where magnification reduces the overall useable portion of content.</li>
```
Comment (alastc) [inline understanding/22/focus-not-obscured-enhanced.html:32]:
```suggestion
    <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) benefit from a clearly visible indication of the current point of keyboard interaction, especially where magnification reduces the overall useable portion of content.</li>
```

File: understanding/22/focus-not-obscured-enhanced.html [hunk 5]
```diff
@@ -21,16 +21,16 @@ <h2>Focus Not Obscured (Enhanced) Success Criterion text</h2>
...
-    <li>People with limited or low vision, who may primarily use a pointer for screen orientation and repositioning, nonetheless benefit from a fuller visible indication of the current point of keyboard interaction, especially where magnification reduces the overall viewing portion of the screen.</li>
-    <li>People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to more easily discover where the focus is located.</li>
+    <li>People with limited or low vision but who rely upon a pointing device (for viewport orientation and repositioning) benefit from a clearly visible indication of the current point of keyboard interaction, especially where magnification reduces the overall useable portion of content.</li>
+    <li>People with attention limitations, short term memory limitations, or limitations in executive processes &mdash; by being able to more easily discover where the focus is located.</li>
```
Comment (alastc) [inline understanding/22/focus-not-obscured-enhanced.html:33]:
```suggestion
    <li>People with attention limitations, short term memory limitations, or limitations in executive processes benefit by being able to more easily discover where the focus is located.</li>
```

**Pull request merged**